src/Core/Types/Unifier.cs

```diff
@@ -581,7 +588,44 @@ private bool TryMakeFieldName(StructureField fa, StructureField fb, out string n
             return true;
         }
 
-		public DataType UnifyPointer(Pointer ptrA, DataType b)
+        private bool IsNestedStructure(
+            StructureField fa, StructureField fb,
+            out StructureField fNestedStruct, out StructureField fOther)
+        {
+            var strFa = fa.DataType.TypeReferenceAs<StructureType>();
```

Comment:
I'm not too keen on multiple `out` parameters. It seems like a better name for this would be `FindNestedStructure` with either a `Tuple<StructureField,StructureField>` or a new class `NestedStructureFields` as a return type.

I'm also having a little trouble following the logic in the method. Perhaps a few judicious comments would clarify?